frontend/src/queries/schema.ts

```diff
@@ -786,7 +786,7 @@ export type InsightFilter =
 export interface InsightActorsQuery {
     kind: NodeKind.InsightActorsQuery
     source: InsightQueryNode
-    day?: string
+    day?: string | number
```

Comment:
We should make sure that number will be turned into integer.

Not sure with that union, maybe something like this is required:

```ts
/** @asType integer */
export type Day = number

export interface Bla {
    day?: string | Day
    ...
}
```

Reply:
You mean after [this PR](https://github.com/PostHog/posthog/pull/19408) goes in, the above will likely need special handling?

Reply:
Regardless of the mentioned PR, we should annotate so that numbers become integers in Python, unless we need float specifically.
As for the PR, if we surface the types in docs, my workaround might indeed look weird. Maybe there is another way. Can't we put the number in the string, thus keep just string, and call it a day? 😅

Reply:
Yep, agreed on `int` - have updated for the time being

Reply:
Can we just use another field? The lifecycle query uses `day` for its `YYYY-MM-DD` date. Can we use `dayIndex` for stickiness instead?